packages/renderer/src/components/GetOtp.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import styles from './GetOtp.module.css';
+import { Button, FormControl, Modal } from 'react-bootstrap';
+import { observer } from 'mobx-react-lite';
+import { useConfigStore } from '/@/store/ConfigStore';
+import { sendUserInput } from '#preload';
+import { ModalStatus } from '/@/types';
+
+const GetOtp = () => {
+  const configStore = useConfigStore();
+  const [modalStatus, setModalStatus] = useState<ModalStatus>(ModalStatus.HIDDEN);
+  const [inputText, setInputText] = useState('');
+
+  const closeModal = () => setModalStatus(ModalStatus.HIDDEN);
```

Comment:
The `ModalStatus` is to reuse the main modal (`Body.tsx`) for different usages.

Here you create a new Modal, I think you can just use a boolean state to decide if the Modal is open or not.

Reply:
Done. changed to boolean